helper-cli/src/main/kotlin/commands/classifications/ImportCommand.kt

```diff
@@ -0,0 +1,121 @@
+        help = "The name of the provider to import license classifications from. Must be one of " +
+                "${enumValues<LicenseClassificationProvider>().map { it.name }}."
+    ).enum<LicenseClassificationProvider>()
+
+    // TODO: Make this default to the provider name with clikt 4, see https://github.com/ajalt/clikt/issues/381.
+    private val prefix by option(
+        "--prefix", "-p",
+        help = "A prefix to use for all category names declared by the provider's classifications."
+    )
+
+    private val licenseClassificationsFile by option(
+        "--output", "-o",
+        help = "The file to write the provided license classifications to."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+
+    override fun run() {
+        val classifications = provider.getClassifications().run {
+            prefix?.let { prefixCategoryNames(it) } ?: this
+        }
+
+        val yaml = yamlMapper.writeValueAsString(classifications)
+
+        licenseClassificationsFile?.run {
+            writeText(yaml)
+        } ?: println(yaml)
+    }
+}
+
+private data class EclipseLicenses(
+    val meta: Map<String, String>,
+    val approved: Map<String, String>,
+    val restricted: Map<String, String>
+)
+
+private enum class LicenseClassificationProvider(val url: String) : Logging {
+    ECLIPSE("https://www.eclipse.org/legal/licenses.json") {
+        override fun getClassifications(): LicenseClassifications {
+            val json = jsonMapper.readValue<EclipseLicenses>(URL(url))
+
+            logger.info { "Importing Eclipse license classifications dated ${json.meta["updated"]}." }
+
+            val approved = LicenseCategory("approved")
+            val restricted = LicenseCategory("restricted")
+            val categorizations = mutableListOf<LicenseCategorization>()
+
+            json.approved.mapTo(categorizations) { (id, _) ->
+                LicenseCategorization(SpdxSingleLicenseExpression.parse(id), setOf(approved.name))
+            }
+
+            json.restricted.mapTo(categorizations) { (id, _) ->
+                LicenseCategorization(SpdxSingleLicenseExpression.parse(id), setOf(restricted.name))
+            }
+
+            return LicenseClassifications(
+                categories = listOf(approved, restricted),
+                categorizations = categorizations
+            )
+        }
+    },
+    LDBCOLLECTOR("https://github.com/maxhbr/LDBcollector/raw/generated/ort/license-classifications.yml") {
+        override fun getClassifications(): LicenseClassifications = yamlMapper.readValue(URL(url))
+    };
+
+    abstract fun getClassifications(): LicenseClassifications
+}
+
+private fun LicenseClassifications.prefixCategoryNames(prefix: String) =
+    LicenseClassifications(
+        categories = categories.map { it.copy(name = prefix + it.name) },
```

Comment:
Do we commonly use string interpolation / should this be done here as well? (Same below)

Reply:
I believe we only use / prefer string interpolation when mixing with literal stings.